src/components/quote/Quote.types.ts

```diff
@@ -0,0 +1,8 @@
+import { SpaceProps, LayoutProps } from 'styled-system'
+
+export interface QuoteProps
+  extends React.HTMLAttributes<HTMLQuoteElement>,
+    SpaceProps,
+    LayoutProps {
+  children: string
```

Comment:
We don't need the `children` here as it is covered by the `React.FC<...>` in `Quote.tsx`.